docs/docs-components/components/navigation.html

```diff
@@ -13,6 +13,30 @@
 
 <body>
     <div class="documentation-page">
+        <div class="mobile-nav px-32">
+            <i class="fas fa-times close-btn position-absolute">
+            </i>
+            <ul class="list-style-none  my-32">
+            <li class="py-8"><a href="/index.html">Home</a></li>
+            <li><a href="">Installation</a></li>
```

Comment:
You should keep `the href="#"` or you can remove the `<a>` if you don't want it to redirect anywhere.

Reply:
Made the changes please check.